pages/dkp/kaptain/2.0.0/install/dkp/index.md

```diff
@@ -137,8 +137,7 @@ If you added Kaptain after installing DKP, you must make it available by creatin
 
 ## Deploy Kaptain on selected Workspaces
 
-You have installed Kaptain by adding it to the DKP Catalog applications. The next step is to enable and deploy Kaptain on all clusters in a selected Workspace. For this, refer to [Deploy Kaptain] instructions. 
-<!-- Need to add link to this topic once it is created -->
+You have installed Kaptain by adding it to the DKP Catalog applications. The next step is to enable and deploy Kaptain on all clusters in a selected Workspace. For this, refer to [Deploy Kaptain][deploy] instructions.
```

Comment:
This works for DKP 2.2. In DKP 2.1 we have so many extra air gap steps I think we should just use the helm install.